CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+This file provides general guidance for developers that are creating or updating REST API definitions for Azure.
+
+## Table of Contents
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      markdown-toc -i --maxdepth 4 CONTRIBUTING.md
+  -->
+
+<!-- toc -->
+
+- [Design Guidelines](#design-guidelines)
+- [Coding Style](#coding-style)
+- [Directory Structure](#directory-structure)
+- [Pull Requests](#pull-requests)
+- [Adding a new service](#adding-a-new-service)
+- [Running tests](#running-tests)
+  * [Unit tests](#unit-tests)
+  * [Integration tests](#integration-tests)
+- [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+- [Additional Resources](#additional-resources)
+
+<!-- tocstop -->
+
+## Reporting problems
+
+If you discover a problem with a REST API document in this repo, feel free to [open an issue](https://github.com/Azure/azure-rest-api-specs/issues/new). But please do not report issues with service behavior / functionality in this repo.
+
+When opening an issue, please clearly describe the problem including which REST API definition is in error and what elements of the definition are incorrect or missing.
+
+## Avoid Breaking Changes
+
+The first rule for Azure REST APIs is to avoid breaking changes in GA'd APIs. The [Azure Breaking Changes Policy](aka.ms/AzBreakingChangesPolicy) is a formal description of what changes are considered breaking changes. This rule trumps adherence to Design Guidelines, Coding Style, etc.
```

Comment:
I'd define acronyms the first time - at least per H2.
```suggestion
The first rule for Azure REST APIs is to avoid breaking changes in Generally Available (GA'd) APIs. The [Azure Breaking Changes Policy](aka.ms/AzBreakingChangesPolicy) is a formal description of what changes are considered breaking changes. This rule trumps adherence to Design Guidelines, Coding Style, etc.
```